sdks/python/apache_beam/examples/fastavro_it_test.py

```diff
@@ -181,7 +175,7 @@ def assertEqual(l, r):
           if l != r:
             raise BeamAssertException('Assertion failed: %s == %s' % (l, r))
 
-        assertEqual(v.keys(), ['avro', 'fastavro'])
+        assertEqual(list(v.keys()), ['avro', 'fastavro'])
```

Comment:
We should use `sorted` instead of `list` if tests run in Python 2.7/3.5/3.6.

Reply:
Good point, fixed. Thanks! PTAL.